src/systems/systems/src/navigation/adirs.rs

```diff
@@ -2261,6 +2380,48 @@ mod tests {
             assert_eq!(test_bed.track(adiru_number).normal_value().unwrap(), angle);
         }
 
+        #[rstest]
+        #[case(1)]
+        #[case(2)]
+        #[case(3)]
+        fn drift_angle_is_supplied_when_ground_speed_greater_than_or_equal_to_50_knots(
+            #[case] adiru_number: usize,
+        ) {
+            let heading = Angle::new::<degree>(160.);
+            let track = Angle::new::<degree>(180.);
+            let mut test_bed = all_adirus_aligned_test_bed_with()
+                .track_of(track)
+                .and()
+                .heading_of(heading)
+                .and()
+                .ground_speed_of(Velocity::new::<knot>(
+                    InertialReference::MINIMUM_GROUND_SPEED_FOR_TRACK_KNOTS,
+                ));
```

Comment:
**nitpick**: The goal of `and` here is simply to form English sentences. So probably you'd want to only keep the last one:
> All adirus aligned test bed with track of 123, heading of 124, and ground speed of 50.